themes/tiddlywiki/tight/base.tid

```diff
@@ -105,13 +105,13 @@ tags: [[$:/tags/Stylesheet]]
 
 	html .tc-sidebar-resizer {
 		width: 12px;
-		left: clamp(calc({{$:/themes/tiddlywiki/vanilla/metrics/storyleft}} + {{$:/themes/tiddlywiki/vanilla/metrics/storyminwidth}}),clamp(calc({{$:/themes/tiddlywiki/vanilla/metrics/storywidth}}),max(calc({{$:/themes/tiddlywiki/vanilla/metrics/storyleft}} + {{$:/themes/tiddlywiki/vanilla/metrics/storywidth}}),calc({{$:/themes/tiddlywiki/vanilla/metrics/storyright}})),max(calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarminwidth}} + ({{$:/themes/tiddlywiki/vanilla/metrics/storyleft}} + {{$:/themes/tiddlywiki/vanilla/metrics/storywidth}} - {{$:/themes/tiddlywiki/vanilla/metrics/storyright}})),calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarminwidth}}))),calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarminwidth}}));
+		left: clamp(calc({{$:/themes/tiddlywiki/vanilla/metrics/storyleft}} + {{$:/themes/tiddlywiki/vanilla/metrics/storyminwidth}}),clamp(calc({{$:/themes/tiddlywiki/tight/metrics/storywidth}}),max(calc({{$:/themes/tiddlywiki/vanilla/metrics/storyleft}} + {{$:/themes/tiddlywiki/tight/metrics/storywidth}}),calc({{$:/themes/tiddlywiki/tight/metrics/storyright}})),max(calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarminwidth}} + ({{$:/themes/tiddlywiki/vanilla/metrics/storyleft}} + {{$:/themes/tiddlywiki/tight/metrics/storywidth}} - {{$:/themes/tiddlywiki/tight/metrics/storyright}})),calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarminwidth}}))),calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarminwidth}}));
 	}
 
 	<%if [{$:/themes/tiddlywiki/vanilla/options/sidebarlayout}match[fluid-fixed]] %>
 
 		html .tc-sidebar-resizer {
-			left: clamp(calc({{$:/themes/tiddlywiki/vanilla/metrics/storyleft}} + {{$:/themes/tiddlywiki/vanilla/metrics/storyminwidth}}),calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarwidth}}),calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarminwidth}}));
+			left: clamp(calc({{$:/themes/tiddlywiki/vanilla/metrics/storyleft}} + {{$:/themes/tiddlywiki/vanilla/metrics/storyminwidth}}),calc(100% - {{$:/themes/tiddlywiki/tight/metrics/sidebarwidth}}),calc(100% - {{$:/themes/tiddlywiki/vanilla/metrics/sidebarminwidth}}));
 		}
 
 	<% endif %>
```

Comment:
Does this mean, if I install a user developed theme, then I have to edit it like above to support the sidebar resizer?

Reply:
Hi @kookma - I don't think it's possible to make the resizer compatible with all possible themes out of the box
That's because one theme uses these metrics, another theme uses different metrics, one theme has the sidebar at the left, the other at the right...

Reply:
But seems Gk0Wk's JS version sidebar-resizer can adapt to every themes? At least itonnote theme haven't done any adaption to it. Of course it has limintation, like it force the unit to be `vw`.

Reply:
Here it was asked to change the theme metrics directly. If a theme uses different metrics tiddlers than the vanilla ones it becomes a bit more complex. But I think we could make it easier for other themes to adapt the resizer.

Reply:
@linonetwo - having a look at Gk0Wk's JS resizer at https://gk0wk.github.io/TiddlySeq/#%24%3A%2Fplugins%2FGk0Wk%2Fsidebar-resizer%2Fhook.js I doubt that it can work with fixed-fluid vanilla mode for example. In fact, on the demo page try to set the mode to fixed-fluid... It changes the sidebarwidth only and does not adapt to other themes or theme-options.

Reply:
> I don't think it's possible to make the resizer compatible with all possible themes out of the box

Thank you for the clarification! So, some default core themes will support the sidebar resizer.

Reply:
Hi @kookma , please don't get me wrong. My intention is to make it possible for other themes to adapt the resizer. I will just have to think about it for a while

Reply:
> Thank you for the clarification! So, some default core themes will support the sidebar resizer.

All default core themes support the resizer at the moment.
I'm thinking about other themes that have the sidebar at the left for example. Those themes would need to set a specific config tiddler to yes and we could make the resizer adapt to it.
But I believe themes must use the metrics of the currently selected theme, otherwise this isn't possible.
Please ask if you have any questions